routers/api/v2/tokens.go

```diff
@@ -64,3 +66,33 @@ func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
 		Token: token,
 	})
 }
+
+// DELETE: /api/v2/tokens/service/:id
+// Response:
+// Headers:  Authorization <- token
```

Comment:
```suggestion
// Headers:  Authorization -> token
```

since it's extracting the token from the header